### DISCUSSION TITLE
江蘇男與價值9萬元「AI性愛機器人」同居一年多　花1萬為她做SPA

### DISCUSSION THREAD

**Discussion by luo910227** [1101-GS3073-C/Python-Community-GS3073-C#389] [科技新知]
「2017年，AI女性愛機器人Harmony誕生，2018年，世界上第一個男性愛機器人Henry問世，這些有「思想」（植入了人工智能）的性愛娃娃，一上線便迅速售罄。女性愛機器人Harmony皮膚由矽膠製作，手感完美，還懂得傾聽、交流……Harmony在一些小圈子裡得到爆炸性討論。幾條網友的評論尤為顯眼：「性需求在全球來說，都是隱秘而洶湧的。」「有了她，誰還去戀愛，甚至結婚？」 」

性愛機器人議題是值得討論的，現在疫情爆發減少了人與人的互動，也可能增加性愛機器人的需求。不過此完成需求的方式是否牽涉到一些道德倫理的問題?或是，性愛機器人AI技術是否有權利拒絕?又或者此方式會不會對使用者造成某方面的心理影響?

資料來源：https://reurl.cc/zZ4QYQ

**Comment by Joan0518** [comment]
好新聞的標題哈哈哈

**Comment by AngusGuei** [comment]
現今社會 越來越多人不去認識或接觸新的朋友圈，或許這種人性化的機器人以後會變成趨勢？

**Comment by YuHengChen54** [comment]
感覺對於少部分的人這個真的是蠻好的東西，不過家裡放著一個不是人的人我覺得蠻恐怖的哈哈哈

**Comment by brian431** [comment]
> 「2017年，AI女性愛機器人Harmony誕生，2018年，世界上第一個男性愛機器人Henry問世，這些有「思想」（植入了人工智能）的性愛娃娃，一上線便迅速售罄。女性愛機器人Harmony皮膚由矽膠製作，手感完美，還懂得傾聽、交流……Harmony在一些小圈子裡得到爆炸性討論。幾條網友的評論尤為顯眼：「性需求在全球來說，都是隱秘而洶湧的。」「有了她，誰還去戀愛，甚至結婚？」 」
> 
> 
> 
> 性愛機器人議題是值得討論的，現在疫情爆發減少了人與人的互動，也可能增加性愛機器人的需求。不過此完成需求的方式是否牽涉到一些道德倫理的問題?或是，性愛機器人AI技術是否有權利拒絕?又或者此方式會不會對使用者造成某方面的心理影響?
> 
> 
> 
> 資料來源：https://reurl.cc/zZ4QYQ

感覺是個非常有趣的科技，沒有什麼拒絕的理由，不過確實挺有病的。

**Comment by Chenszuhong** [comment]
這是個有趣的科技，但是感覺會脫離人與人的連結

**Comment by xCk27x** [comment]
感覺會出現這樣的需求
是因為太多人即使擁有良好經濟條件
卻過於內向 或不知如何拓展社交圈
難怪別人都說大學必修戀愛學分XD

**Comment by BingRong1217** [comment]
很不錯的科技發展，但是否會衍生出相關問題

**Comment by skyling0299** [comment]
AI人工倫理的通識課上有提到性、愛與陪伴，雖然還沒上到那堂課，但對AI是否能取代真人也感到好奇

**Comment by Ben6uo** [comment]
使用 AI 性愛機器人也有一些倫理問題需要考慮

**Comment by limitx0** [comment]
不知道以後會不會立法讓人和機器人結婚

**Comment by awehao** [comment]
性癖可以冷門，不能邪門

**Comment by Aerofe** [comment]
現在的科技真的讓人與人的接觸越來越少了😂

**Comment by Raywei0214** [comment]
所謂性愛機器，關鍵是性愛機器需要如何解讀。逸經講過一段耐人尋思的話，不要對每一個故事都信以為真。這句話改變了我的人生。如果別人做得到，那我也可以做到。

**Comment by awkward-willy** [comment]
在疫情的時代下，隨著人們與這些機器相處的時間逐漸拉長，也會漸漸產生情愫，單論陪伴與解決性需求的層面來說，這樣的機器確實是相對快速且不易產生傳染病等問題。但或許從另一個角度來說，是否意味著現代人對於和其他人的互動感到沒自信或沒興趣呢？

**Comment by larry40305** [comment]
如果真的實現的話，少子化要越來越嚴重了
